contracts/Standards/ITokenWithDeposit.sol

```diff
@@ -1,33 +0,0 @@
-pragma solidity 0.4.15;
-
-import './IERC20Token.sol';
-
-
-contract ITokenWithDeposit is IERC20Token {
```

Comment:
Why should we delete it? Then you duplicate definitions for log events and withdrawal. Is this because of `deposit` that is payable?

Reply:
Yes, EuroToken's deposit function is too different from EtherToken's. Different people call it, payable, etc. Also, as an interfaec, what does it mean to have a `TokenWithDeposit`? Does it mean I can now call `deposit` on it? What do I pay it with?

Reply:
It simply would be nice to have interface to tokens where you can deposit and withdraw assets. pity there's too many differences. I'm still for some kind of standard here. Let's merge now. I will think about it :)